### PR TITLE
Enable PInvoke stub compilation for R2R

### PR DESCRIPTION
**Pull request by clamp03**

### CODE REVIEW

**Comment by clamp03** [comment]
I fixed to enable IL STUB compilation in crossgen for ARM target on ARM device.
I think that test failed because this patch generates different files on ARM and X64.

**Comment by jkotas** [comment]
> I think that test failed because this patch generates different files on ARM and X64.

Yes, that's the problem. Do you plan to do something about it?

**Comment by clamp03** [comment]
@jkotas Sorry for the late response. I couldn't work for weeks because of COVID19.
I want to make a commit that generates STUB in case of cross bitness next week.
Thank you.

**Comment by clamp03** [comment]
I think a failure on Windows_NTx86 is not by this patch. I will check a failure on Linux arm64.
Thank you.

**Comment by clamp03** [comment]
@jkotas  I think test failures are not by this patch. I don't know how to manage issues. Could you help?

File: src/coreclr/src/vm/method.cpp [hunk 1]
```diff
@@ -1329,11 +1329,11 @@ ReturnKind MethodDesc::ParseReturnKindFromSig(INDEBUG(bool supportStringConstruc
 
 ReturnKind MethodDesc::GetReturnKind(INDEBUG(bool supportStringConstructors))
 {
-#ifdef HOST_64BIT
+#ifdef TARGET_64BIT
```
Comment (jkotas) [inline src/coreclr/src/vm/method.cpp:1332]: This ifdef can be just delete (keep the comment though).

**Review by jkotas — APPROVED** [review]

File: src/coreclr/src/vm/method.cpp (hunk 1 above)
Comment (clamp03) [reply src/coreclr/src/vm/method.cpp]: Fixed. Thank you.

**Comment by jkotas** [comment]
Thanks!

**Comment by clamp03** [comment]
I squashed and made to one commit. And all checkes are passed.
Could you please review or merge? Thank you.

**Comment by clamp03** [comment]
@jkotas Thank you!

**Pull request merged**